mathesar/views/export.py

```diff
@@ -71,6 +79,55 @@ def stream_table_as_csv(
     return response
 
 
+def dump_schema(
+    request,
+    database_id: int,
+    schema_oid: int
+) -> FileResponse:
+    user = request.user
+    with connect(database_id, user) as conn:
+        schema_name = get_schema(schema_oid, conn)['name']
+        pg_dump_cmd = [
+            'pg_dump',
+            '-h', conn.info.host,
+            '-p', str(conn.info.port),
+            '-U', conn.info.user,
+            '-d', conn.info.dbname,
+            '-n', sql.Identifier(schema_name).as_string(),
+            '-O'  # Don't include owner info in the dump
```

Comment:
I think you should add the `--inserts` flag here. We should prioritize compatibility over performance, unless the performance really becomes a problem for someone.

I'd also like you to look into whether there's a flag to add that will get rid of the `SET` commands at the start of the file. Those can also cause problems when loading, and aren't usually needed.

Reply:
> I'd also like you to look into whether there's a flag to add that will get rid of the SET commands at the start of the file.

There is no such flag, unfortunately, users would have to take those statements out manually. Apparently, postgres includes those statements to make sure that upon restore, the server behaves exactly the same as when the dump was created. 

Given this, and the fact that we use `GENERATED BY DEFAULT AS IDENTITY` for `id` columns, would it make sense for us to prioritize compatibility over performance? The performance for someone dumping their local database and restoring it on a remote server would be terrible if there's a lot of data. Assuming this is a fairly common use case for someone using our product.